src/main/java/com/vi/migrationtool/tenantservice/ImportDiocesesFromAgencyToTenantServiceTask.java

```diff
@@ -0,0 +1,58 @@
+package com.vi.migrationtool.tenantservice;
+
+import com.vi.migrationtool.common.MigrationTasks;
+import com.vi.migrationtool.config.BeanAwareSpringLiquibase;
+import liquibase.database.Database;
+import lombok.Data;
+import lombok.extern.slf4j.Slf4j;
+import lombok.val;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+@Slf4j
+public class ImportDiocesesFromAgencyToTenantServiceTask extends MigrationTasks {
+
+    private static final String CREATE_DIACOSE_TABLE = "CREATE TABLE IF NOT EXISTS diocese (id BIGINT NOT NULL AUTO_INCREMENT, name VARCHAR(255) NOT NULL, PRIMARY KEY (id) UNIQUE (name));";
```

Comment:
I think that CREATE diocese should be migration that is part of TenantService (so that TenantService is aware of it's tables), but the data migration itself needs to be implemented in the migrationTool